stories/Button/Button.stories.js

```diff
@@ -18,7 +18,7 @@ export default {
   // More on argTypes: https://storybook.js.org/docs/html/api/argtypes
   argTypes: {
     scaleMultiplier: {
-      name: "active scale multiplier / 1000 (744 = 74.4%)",
+      name: "active scale multiplier / 1000 (744 = 0.744%)",
       defaultValue: 1000,
       control: { type: "range", min: 700, max: 1000 },
```

Comment:
Instead of changing it to whole integers, could you try changing the control `step` to `0.1`? I haven't tried it, but I feel like it should work.

`{ control: { type: "range", min: 70, max: 100, step: 0.1 } }}`

details on this control on the [storybook docs](https://storybook.js.org/docs/react/essentials/controls#annotation).

Reply:
that does work, let me make the update